recipe/meta.yaml

```diff
@@ -1,11 +1,9 @@
-{% set lab_spec = ">=2,<3.0.0a0" %}
-{% set py_spec = ">=3.5" %}
 {% set name = "jupyter-lsp" %}
 {% set ext_name = "@krassowski/jupyterlab-lsp" %}
-{% set version = "0.9.2" %}
-{% set build_number = 1 %}
+{% set version = "0.9.3" %}
+{% set build_number = 0 %}
 
-{% set py_spec = ">=3.5" %}
+{% set py_spec = ">=3.6" %}
 {% set lab_spec = ">=2.2,<3.0.0a0" %}
 {% set ext_spec = "@^2.0.0" %}
```

Comment:
2.1.0?

Reply:
As far as I understood it is not mandatory. But if you think it is safer, I'll update it too.